misc/password.go

```diff
@@ -0,0 +1,44 @@
+package misc
+
+import (
+	"fmt"
+	"github.com/labbsr0x/goh/gohtypes"
+	"net/http"
+	"strings"
+)
+
+const (
+	passMinCharacters       = 12
+	passMaxCharacters       = 30
+	passMinUniqueCharacters = 7
+)
```

Comment:
@abilioesteves should I add these constants as flags of the program with these values as defaults?

Reply:
Nope. I think we have to work in the realm of password policies, not password rules. Right now we have only one policy. When there is a request for more policies, we reorganize the code to work with a password policy parameter.

Reply:
OK